src/main/java/seedu/address/model/ResumeBook.java

```diff
@@ -517,6 +529,64 @@ public int getResumeSize() {
         return resumes.getSize();
     }
 
+
+    //=========== Notes ================================================================================
+
+    /**
+     * Returns true if a skill with the same identity as {@code skill} exists in the resume book.
+     */
+    public boolean hasNote(Note note) {
+        requireNonNull(note);
+        return notes.contains(note);
+    }
+
+    /**
+     * Adds a skill to the resume book.
+     * The skill must not already exist in the resume book.
+     */
+    public void addNote(Note note) {
+        requireNonNull(note);
+        notes.add(note);
+    }
+
+    /**
+     * Replaces the given skill {@code target} in the list with {@code editedSkill}.
+     * {@code target} must exist in the resume book.
+     * The identity of {@code editedSkill} must not be the same as another existing skill in the resume book.
+     */
+    public void setNote(Note target, Note editedNote) {
+        notes.setItem(target, editedNote);
+    }
+
+    /**
+     * Removes {@code key} from this {@code ResumeBook}.
+     * {@code key} must exist in the resume book.
+     */
+    public void deleteNote(Note key) {
+        int id = key.getId();
+        notes.remove(key);
+    }
+
+    @Override
+    public Note getNote(Index index) {
+        return notes.asUnmodifiableObservableList().get(index.getZeroBased());
+    }
+
+    @Override
+    public int getNoteListSize() {
+        return notes.getSize();
+    }
+
+    @Override
+    public UniqueItemList<Item> getPersonalDetailList() {
+        return null;
+    }
+
+    @Override
+    public Note getNoteByIndex(int id) {
```

Comment:
I guess you left this empty to push first? Making comment so we don't forget to fix